lib/scholar/linear/svm.ex

```diff
@@ -0,0 +1,257 @@
+defmodule Scholar.Linear.SVM do
+  @moduledoc """
+  SVM classifier
+
+  It uses the OvR strategy to handle both binary and multinomial classification.
+  This implementation uses stochastic gradient descent from default or any other optimizer
+  available in `Polaris`. This makes it similar to a sklearn SGDClassifier [1].
+  It means that on average it will work slower than algorithms that use QP and kernel trick (LIBSVM [2]) or
+  Coordinate Descent Algorithm (LIBLINEAR [3]). It also cannot use different kernels like in LIBSVM,
+  but you can use any type of optimizer and regularizer available in `Polaris`.
```

Comment:
Polaris doesn't have a separate concept of regularizers. Perhaps this should be removed.
```suggestion
  but you can use any type of optimizer available in `Polaris`.
```